### ISSUE TITLE
How to understand page and pages

### ISSUE THREAD

**Issue by dittaeva** [citation-style-language/locales#145]
"page" and "pages" have been translated to "side" and "sider" in Norwegian. However, "sider" is the  indefinite plural form of "side". Won't the use in CSL always be definite? I.e. it will always be used in connection with a definite range of pages?

The definite plural form is "sidene".

I've requested feedback from fellow translator @unhammer on the issue.

**Comment by unhammer** [comment]
In a sentence like "this is from pages 100 through 103", we would use "sidene".
In a sentence like "this book has 103 pages" we would use "sider". 

I see there are two entries:

```
    <term name="page">
      <single>side</single>
      <multiple>sider</multiple>
    </term>
    <term name="number-of-pages">
      <single>side</single>
      <multiple>sider</multiple>
    </term>
```
– could it be that these correspond to "pages 100-103" and "100 pages", respectively?

**Comment by dittaeva** [comment]
Indeed, I forgot the second use. @aurimasv added "number-of-pages" and should be able to answer your question, but it seems to me to be the reasonable explanation.

**Comment by adam3smith** [comment]
> could it be that these correspond to "pages 100-103" and "100 pages", respectively?

Yes, that's right. We'll gladly take a pull request to fix this accordingly in the Norwegian. It might not work well in localized styles, since most other languages don't have that distinction and we don't use number-of-pages as a term in any style I'm familiar with.

**Comment by unhammer** [comment]
Swedish might want "sidorna" there

**Comment by rmzelle** [comment]
> we don't use number-of-pages as a term in any style I'm familiar with.

Found one! https://github.com/citation-style-language/styles/blob/06764a66c20f06e8b01e0272513161b145e2bfcf/cultivos-tropicales.csl#L191

But yeah, the term is rarely used.

**Comment by dittaeva** [comment]
I've now made a pull request for Norwegian. @torfeur might want to take a look at following @unhammer's suggestion.

**Comment by torfeur** [comment]
Thanks for inviting me! But I fail to se any real life usage that would motivate a change for Swedish - or Norwegian for that matter. If you change "pages" from "sider" to "sidene" you should really change all the terms in the entire long-ordinals - section as well, since all of them are in the indefinite form. Same goes for many other languages. I can understand your point, but if you look at any (all?) other locales they do the same thing (omit the definite article " en - "The..."/ de-De "Die.." /fr-FR "Les.." for "pages/Seiten/pages"). The fact that the Scandinavian languages attaches the definite article of a noun as a suffix does not motivate the inconstancy that "sidene" would become within the Norwegian xml, or in comparison with the other locales.

**Comment by dittaeva** [comment]
I think it is quite irrelevant whether the xml is internally inconsistent or how it is in other languages for the user. I am pretty sure "Sider 12–20" will shout "WRONG" to those who care for localised citation, and they will never have glance at the xml.

Anyway, as far as I can understand, the long ordinals in nynorsk and bokmål are the same whether definite or indefinite, but perhaps you imply "den" should be added, or it is different in Swedish.

**Comment by adam3smith** [comment]
Consistency matters insofar as localization depends on the terms actually being analogous. But I don't fully understand the issue here -- could you maybe clarify for a non-Scandinavian speaker?

**Comment by dittaeva** [comment]
@unhammer described it well (so I'll adapt what he wrote before):
 - multiple of term name "page": In a sentence like "this is from pages 100 through 103", we (in Norwegian) would use "sidene".
 - multiple of term name "number-of-pages": In a sentence like "this book has 103 pages" we (in Norwegian) would use "sider".

Since there is a definite range of pages "sidene" is definite. In English you'd have to add "the" to make it definite, which isn't customary nor likely to be practical, but then I suppose "pages" isn't *not definite* either, it is just lacking of "the".

**Comment by unhammer** [comment]
And to give an example of how wrong it sounds if we have the wrong form, `Dette er frå sider 100–120` sounds about as bad as `She wrote first page and I wrote second page.` does in English.

**Comment by torfeur** [comment]
@adam3smith - I think the main problem is deciding between a literal translation and a usable one.
The literal translation of "pages" is "sider" in norwegian. That translation is however of little practical value - just as @unhammer and @dittaeva  argues. Hence the suggested "sidene" ("the pages")

@dittaeva & @unhammer - you're argument for braking consistency between the different locales is one of usability. I'm with you on that one. @dittaeva - However you have given no real life, end user example of benefits from the suggested change. Aren't you fixing a problem that has (not yet?) surfaced?

**Comment by adam3smith** [comment]
we definitely go for usability - consistency doesn't really make sense (consistency of what?). Given that there are no costs to a better translation, l'd take it even without existing example of use.

**Comment by rmzelle** [comment]
So, just to confirm, "sidene" makes more sense than "sider" for the contexts in which CSL terms are used?

(such as in-text cites with locators, e.g. "(Dahl 1964, sidene 25-27)", or bibliographic entries, e.g. "R. Dahl (2005), Charlie and the Chocolate Factory, i Collected Works of Roald Dahl (sidene 250-546).")

**Comment by unhammer** [comment]
Yeah, though in that context, singular indefinite `side` is maybe even better: `(Dahl 1964, side 25–27)`

**Comment by rmzelle** [comment]
> in that context, singular indefinite `side` is maybe even better

If that's the case, we could also just do:

```xml
    <term name="page">
      <single>side</single>
      <multiple>side</multiple>
    </term>
    <term name="number-of-pages">
      <single>side</single>
      <multiple>side</multiple>
    </term>
```

(it's also worth keeping in mind that individual styles can always overwrite these translations. With the CSL locale files, we're primarily concerned about styles that haven't been specifically localized to Norwegian)

**Comment by dittaeva** [comment]
It is only term name "page" in which singular indefinite can be used (and is better). But yes, that is probably the best solution.

**Comment by rmzelle** [comment]
I just merged #164. Thanks all for your input!
